lib/exp-golomb.js

```diff
@@ -53,7 +53,7 @@ muxjs.ExpGolomb = function(workingData) {
       workingBitsAvailable -= count;
     } else {
       count -= workingBitsAvailable;
-      skipBytes = count / 8;
+      skipBytes = Math.floor(count / 8);
```

Comment:
Yes! But the issue was even more insidious.

The difficulty was that the error was transient because it only happened when `skipBits` crosses a "word" boundary (32-bit block.) What happens is that `skipBytes` is set to a float and `count` in the very next line (58) becomes `0`. 

While the first point means that `workingBytesAvailable` becomes a float, I am not sure that this was causing us any problems since we do not read all of the values in the RBSP.

But the second issue means that every time we crossed a word-boundary, we started reading again at the start of the word (because we never shifted to proper location after the boundary was crossed.)